src/Cardano/Wallet/Network/HttpBridge.hs

```diff
@@ -150,14 +154,25 @@ mkHttpBridge mgr baseUrl network = HttpBridge
         run (map getApiT <$> getEpochById network (EpochIndex ep))
     , getNetworkTip =
         run (blockHeaderHash <$> getTipBlockHeader network)
+    , postSignedTx =
+        -- TODO: We would be good to parse errors from the node
+        -- like "Failed to send to peers: Blockchain protocol error"
+        -- and "Transaction failed verification: transaction has no inputs"
+        toUnit . run . (_postSignedTx network) . ApiT
     }
   where
     run :: ClientM a -> ExceptT HttpBridgeError IO a
     run query = ExceptT $ (first convertError) <$> runClientM query env
+
     env = mkClientEnv mgr baseUrl
+
+    toUnit :: Monad m => ExceptT e m NoContent -> ExceptT e m ()
+    toUnit = mapExceptT (>>= return . fmap (const ()))
+
     getBlockByHash
         :<|> getEpochById
         :<|> getTipBlockHeader
+        :<|> _postSignedTx
```

Comment:
Can we have a similar naming schema for all handlers? `_getNetworkTip` vs `getNetworkTip` seems more honest and less confusing than `getTipBlockHeader ` vs `getNetworkTip` to me.

Reply:
Went with `cPostSigned` etc, as in _client_.